File: blog-sever/server-java/src/main/java/com/site/blog/mapper/ImgMapper.java
package com.site.blog.mapper;

import java.util.List;

import com.baomidou.mybatisplus.core.mapper.BaseMapper;
import com.site.blog.model.entity.Img;

/**
 * @author yanni
 * @date time 2021/10/29 12:46
 * @modified By:
 */
public interface ImgMapper extends BaseMapper<Img> {


  Integer selectCount();
  List<Img> imgs();
}

File: myblog-jpa/src/main/kotlin/com/site/blog/mapper/ImgMapper.kt
package com.site.blog.mapper


import com.site.blog.model.entity.Img
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository


/**
 * @author yanni
 * @date time 2021/10/29 12:46
 * @modified By:
 */
@Repository
interface ImgMapper : JpaRepository<Img,Long>